test/infrastructure/docker/hack/update-deps.sh

```diff
@@ -18,10 +18,10 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-# shellcheck source=./test/infrastructure/docker/hack/utils.sh
-source "$(dirname "$0")/utils.sh"
-# cd to the root path
-cd_root_path
+# shellcheck source=./hack/utils.sh
+source "$(git rev-parse --show-toplevel)/hack/utils.sh"
+# cd to the CAPD root path
```

Comment:
nit: This comment seems pretty useless considering how clear the function name is.

Reply:
This comment seems to be repeated every where. I'll just mention that here since I don't necessarily want to call it out in each file.